integrations/cross_platform/ios.rst

```diff
@@ -0,0 +1,81 @@
+.. _darwin_toolchain:
+
+Using Darwin toolchain package (build require)
+==============================================
+
+Check the `Darwin Toolchain <https://github.com/theodelrieu/conan-darwin-toolchain>`_  package.
+You can use a profile like the following to cross-build your packages for ``iOS``,  ``watchOS`` and ``tvOS``:
+
+.. code-block:: text
+    :caption: ios_profile
+
+    include(default)
+
+    [settings]
+    os=iOS
+    os.version=9.0
+    arch=armv7
+
+    [build_requires]
+    darwin-toolchain/1.0@theodelrieu/stable
```

Comment:
Lets add a note somewhere that this is experimental, and not a package from conan-center-index anymore?